dll/win32/iphlpapi/iphlpapi_fakestubs.c

```diff
@@ -0,0 +1,120 @@
+#include <stubs.h>
+
+// These are here so we can forward the decorated export functions.
+// Without forwarding, we'll lose the decoration
+// See:
+// https://jira.reactos.org/browse/CORE-8462
+// https://jira.reactos.org/browse/CORE-15665
+
+int __stdcall rosstub_PfAddFiltersToInterface(long a0, long a1, long a2, long a3, long a4, long a5)
```

Comment:
Is the `_rosstub` prefix necessary? It seems silly to have to rename them if/when they get implemented in the future.

Reply:
I did not even try it without...

but it seems to work so I'll drop them.